Impossible first try by amateur.

Going through instructions, process fails on
1:
```
error: swiftgodot is required using two different revision-based requirements (fe24cb01640c2d4d48c8555a71adfe346d9543cf and beb2928f404f2905674c5f85f37a63ea2c25eb76), which is not supported
``` 
(Godot 4.3 installed)

2: When I changed revision id to  *beb2928f404f2905674c5f85f37a63ea2c25eb76*
After:
```
cd /home/user/Documents/Code/Swift/KickExampleProject && make all
```
next error appeared:
```
swift build --product Test --build-path ./.build
Downloading binary artifact https://github.com/migueldeicaza/SwiftGodotKit/releases/download/4.3.4/libgodot.xcframework.zip
error: failed downloading 'https://github.com/migueldeicaza/SwiftGodotKit/releases/download/4.3.4/libgodot.xcframework.zip' which is required by binary target 'binary_libgodot': badResponseStatusCode(404)
Downloading binary artifact https://github.com/migueldeicaza/SwiftGodotKit/releases/download/4.3/libgodot.xcframework.zip
error: failed downloading 'https://github.com/migueldeicaza/SwiftGodotKit/releases/download/4.3/libgodot.xcframework.zip' which is required by binary target 'libgodot_tests': badResponseStatusCode(404)
make: *** [build] Error 1
```

PS. Probably I'm doing something wrong. If it's not right way to communicate this, feel free to delete this post and not to answer :) and have a good day!

--- a/Sources/CreateProject/DataFactory.swift
+++ b/Sources/CreateProject/DataFactory.swift
@@ -37,9 +37,9 @@ enum DataFactory {
                     targets: ["\(projectName)"]),
             ],
             dependencies: [
-                .package(url: "https://github.com/migueldeicaza/SwiftGodot", revision: "fe24cb01640c2d4d48c8555a71adfe346d9543cf"),
+                .package(url: "https://github.com/migueldeicaza/SwiftGodot", revision: "a1af0de831a22a2f1d5d8b4221d9df2fdd12978f"),
                 // 4.3.5 tag
-                .package(url: "https://github.com/migueldeicaza/SwiftGodotKit", revision: "1a720b3be1d4b9f87f97c40d03e6ecf735768162")
+                .package(url: "https://github.com/migueldeicaza/SwiftGodotKit", revision: "7f59a1ad97d243a071b548bed7ff573449c82af5")
             ],
             targets: [
                 .executableTarget(

